### DISCUSSION TITLE
How do i update oshi core for minecraft

### DISCUSSION THREAD

**Discussion by Obezzyannki** [oshi/oshi#2679] [Q&A] [answered]
I had same issue like in https://github.com/PaperMC/Paper/issues/7113 and i have no idea how to update oshi core, please tell me, Im not a pro at this stuff. can i have current oshi core as release?

**Comment by dbwiddis** [comment, marked as answer]
If you are building your application from source, you need to add the latest version to the dependencies list in your build system (Maven's `pom.xml` or Gradle's `build.gradle` file).

If you're running an application someone has given you, you need to dig into the packaging to find where the oshi-core JAR file lives, and replace it.
